### DISCUSSION TITLE
trained senter model does not work

### DISCUSSION THREAD

**Discussion by kagoston** [explosion/spaCy#7238] [Help: Coding & Implementations] [answered]
Hi there,
I wanted to train a senter model for my specific text. But after the training it does not work, it does not split the paragraph into sentences.
My steps were:
I have created a '.spacy' file for training and evaluation from 1000 paragraphs
each containing several sentences:

> for i in range(len(sentences)):
> &nbsp;&nbsp;&nbsp;&nbsp; doc = nlp(sentences[i])
> &nbsp;&nbsp;&nbsp;&nbsp; doc[0].is_sent_start = True
> &nbsp;&nbsp;&nbsp;&nbsp; tmp.append(doc)
> trainingDoc = Doc.from_docs(tmp, ensure_whitespace=True)
> db.add(trainingDoc)

I ve created a config file in command line with tok2vec and senter, as in the documentation:
> 	spacy init config config.cfg -p senter -l en

I trained senter component with  'train spacy' in the command line:
	
> spacy train my_config.cfg --paths.train .\trainingFiles\train.spacy --paths.dev .\trainingFiles\dev.spacy 
> 	-o my_model 

I tested it:
> nlp = spacy.load("my_newmodul/model-best")
> text = "some test sentences"
> doc = nlp(text)

the output is the text, not splitted at all

I tried to train it with 1000 sentences, and with 100000 too.
During the training Precision, REcall and F1 are after a few lines 100, Score 1.

Perhaps it would be better to update the en_core_web_sm pretrained model, but I am not sure how should I do it.
Any help would be appreciated.

**Comment by adrianeboyd** [comment, marked as answer]
The description of how you trained the model sounds fine as long as you have training docs that contain multiple sentences.

Just to check: how are you determining whether your `doc` contains multiple sentences? Just `print(doc)` will just show the original text (`doc.text`) without any additional annotation. If you haven't, try looking at `doc.sents` or `token.is_sent_start` to see the sentence boundaries, for example:

```python
for i, sent in enumerate(doc.sents):
  print(i, sent)
```

**Reply by kagoston** [reply]
Thanks for the quick reply.
Yes, you are right!
The problem was that there was a newline at the end of my sentences, so every sentence was a paragraph itself.
Now the model works! Thank you :)
